src/main/kotlin/no/nav/syfo/tilgang/TilgangApi.kt

```diff
@@ -112,5 +113,22 @@ fun Route.registerTilgangApi(
                 )
             }
         }
+
+        post("/system/preloadbrukere") {
+            val callId = call.getCallId()
+            val token = call.getBearerHeader()
+                ?: throw IllegalArgumentException("Failed to preload cache: No Authorization header supplied, callId=$callId")
+
+            // TODO: Max 60 personer i listen?
```

Comment:
En ting @vetlesolgaard har snakket litt om, er at dersom vi lager TODOs i koden, så burde vi lage en oppgave i Trello med litt mer utfyllende info, og så skrive noe a la `TODO: se oppgave IS-1234` i kommentaren. På den måten er det lettere å plukke opp TODOene og gjøre noe med de, sånn at de ikke mister context til nestegang man er i koden og lurer på hva TODOen egentlig handler om.

Det gjelder forsåvidt TODOer som vi ikke skal fikse "med en gang", da, i feks neste PR.

Reply:
Ja, dette var egentlig mest ment for å diskutere og gjøre ferdig nå, bare jeg som glemte å legge inn kommentar da jeg opprettet PRen 😬 

Jeg er enig i at det er lurt å ha en tilhørende lapp, men det er litt risky å skrive en kommentar som ikke har noe annen info enn henvisning til en trello-lapp som kan forsvinne 🤷‍♂️

Reply:
Ja, kanskje et kompromiss da, siden vi arkiverer lapper 🤔 Både forklare hva som må gjøres, samtidig som at man knytter det til en lapp slik at det ikke blir glemt 🙌🏼